### DISCUSSION TITLE
Dec 2023 Review - PR3: Polynomial Long Division

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#45] [Show and tell]
Section: https://teambasedinquirylearning.github.io/precalculus/2024/PR3.html

Author: @sheppardw 

Primary Reviewer: @kathypinzon  (feedback by Dec 11 morning)

**Comment by kathypinzon** [comment]
I would suggest beginning with an activity using just whole numbers to remind students of the long division process.  
What is the quotient and remainder when 346 is divided by 17?
A) quotient 20 remainder 3
B) quotient 20 remainder 6
C) quotient 6 remainder 20
D) quotient 6 remainder 3
Then ask them to in part b to write it as a mixed number.
A) 20 + 3/17
B) 20 + 6/17
C) 6 + 20/17
D) 6 + 3/17

**Comment by kathypinzon** [comment]
I would suggest adding in some scaffolding in a problem before this to get students to understand how the process works.  Something like what you see in #3 of this worksheet https://www.mathcentre.ac.uk/resources/uploaded/mc-ty-polydiv-2009-1.pdf

Begin by asking them how many times does Y go into Z and then MC answers.

**Comment by kathypinzon** [comment]
The question you had written would make a great last question in the module.  I would suggest moving the divisor question to the top though since they need to know and use that before finding the quotient and remainder.
